net-proxy/3proxy/3proxy-0.8.8.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+EAPI=6
+inherit eutils
+
+DESCRIPTION="A really tiny cross-platform proxy servers set"
+HOMEPAGE="http://www.3proxy.ru/"
+SRC_URI="https://github.com/z3APA3A/3proxy/archive/${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~ppc"
+IUSE=""
+
+S="${WORKDIR}/${PN}-${P}"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gentoo.patch
```

Comment:
don't use `epatch` in EAPI=6, stick the patch in global `PATCHES` and let `default` do the patching, you can then get rid of the `inherit eutils`